src/Stratis.Bitcoin.IntegrationTests.Common/TestHelper.cs

```diff
@@ -261,17 +262,21 @@ public TransactionNode(Transaction tx)
         public static void Connect(CoreNode from, CoreNode to)
         {
             from.CreateRPCClient().AddNode(to.Endpoint, true);
```

Comment:
I know this code hasn't change as part of this work, but can the logic check if the nodes are already connected - or wouldn't that really matter ?

Reply:
hmm well as this is a test we should know exactly who is connected (I could add a test ensure not connected perhaps?)